frontend/src/components/teacher/student-management/classroom-summary/ClassroomPopover.tsx

```diff
@@ -112,27 +81,13 @@ const ClassroomPopover = ({
         isOpen={isEditModalOpen}
         onClose={onEditModalClose}
       />
-      <Modal
-        body={
-          <>
-            <Text color="grey.300" pt={2} textStyle="paragraph">
-              Are you sure you want to archive this classroom? You can find your
-              archived classrooms in the &quot;Archived&quot; tab.
-            </Text>
-            <Text color="red.200" pt={2} textStyle="paragraph">
```

Comment:
are we getting rid of this note?

Reply:
I'm honestly undecided. The wording I updated it to is more consistent with other modals; on the other hand, it might still be nice to call attention to the behaviour more noisily. What do you think we should do?

Reply:
i think we should provide this info somewhere, since the behaviour would not otherwise be known. easiest UI fix is in the modal text.. not sure what other options we have